### DISCUSSION TITLE
Logic ID operand limit

### DISCUSSION THREAD

**Discussion by rts18** [iNavFlight/inav#8315] [General]
My inquiry is related to the number of logic ID's that can be accessed under operand A or B.  The limit is 31. While the activator can handle up to 63.
Is this by design to save flash? Or possibly something that has been over looked?

**Comment by MrD-RC** [comment]
Not a problem. All 64 are visible to me. Are you using the correct version of Configurator?

![image](https://user-images.githubusercontent.com/17590174/184496463-236aa463-f418-4a4e-871d-08af6724c7b4.png)

![image](https://user-images.githubusercontent.com/17590174/184496475-64428125-99ab-4f57-bd7a-c88e38f69653.png)

**Comment by MrD-RC** [comment]
Ah, no. I see the problem.

![image](https://user-images.githubusercontent.com/17590174/184498140-65bcb4fc-805e-4c87-84e7-41f15893f758.png)

**Comment by MrD-RC** [comment]
Fixed
https://github.com/iNavFlight/inav-configurator/pull/1603
